src/marketplace/eqp/v1/test-results.md

```diff
@@ -185,6 +185,8 @@ curl -X GET \
 }
 ```
 
+<a id="cpd"/>
```

Comment:
Delete and append {#cpd} to the heading below.

Reply:
Thank you for resolving!